models/src/main/scala/it/pagopa/interop/agreementmanagement/model/agreement/PersistentAgreement.scala

```diff
@@ -19,5 +19,6 @@ final case class PersistentAgreement(
   consumerDocuments: Seq[PersistentAgreementDocument],
   createdAt: OffsetDateTime,
   updatedAt: Option[OffsetDateTime],
-  consumerNotes: Option[String]
+  consumerNotes: Option[String],
+  document: Option[PersistentAgreementDocument]
```

Comment:
This is precisely the agreement document, calling it `agreementDocument` is redundant because it belongs to the Agreement object.
The only other name that comes to my mind is `contract`.
What do you think?

Reply:
`contract` is really nice 👍